src/autolua.zig

```diff
@@ -172,6 +172,12 @@ pub fn check(L: ?*lua.lua_State, idx: c_int, comptime T: type) T {
             // lua.lua_pop(L, 1);
             // const ptr = lua.lua_touserdata(L, idx);
         },
+        .Optional => |N| {
+            if (lua.lua_type(L, idx) < lua.LUA_TBOOLEAN) {
```

Comment:
Why `< LUA_TBOOLEAN` rather than `<= 0` (which is what you see in the lua source: https://www.lua.org/source/5.4/lua.h.html#lua_isnoneornil)

Reply:
i think it's better to use `lua_isnoneornil`